src/checkIfConditionMatches.js

```diff
@@ -59,10 +59,8 @@ export function checkIfConditionMatchesProperties(node, formValues, formFields,
 
     return validateFieldSchema(
       {
-        options: field.options,
-        // @TODO/CODE SMELL. We are passing the property (raw field), but buildYupSchema() expected the output field.
+        ...field,
         ...currentProperty,
-        inputType: field.inputType,
```

Comment:
though: I wonder why we never passed the entire `...fied` before. Perhaps to avoid some other bug? 🤔 

Pipelines are green, both here and the [internally repo](https://gitlab.com/remote-com/employ-starbase/dragon/-/merge_requests/23636), so I'm trusting this won't break something else.

Reply:
i apparently implemented some of this before, and I'm pretty sure I was leaning on the conservative side not wanting to pass down everything unless necessary, and this is how it manifested ... 6+ months later.